sysfs/sysfs_macros/src/lib.rs

```diff
@@ -388,7 +388,8 @@ struct SetterFunction {
 fn let_sysfs_path(sysfs_dir: &Option<LitStr>, sysfs_file: &str) -> Stmt {
     let literal = match sysfs_dir {
         Some(sysfs_dir) => format!("{}/{}", sysfs_dir.value(), sysfs_file),
-        None => format!("{}/{}", "{SYSFS_DIR}", sysfs_file),
+        // TODO what is SYSFS_DIR, where is it supposed to come from?
+        None => format!("{}/{}", "SYSFS_DIR", sysfs_file),
```

Comment:
no idea what constant `SYSFS_DIR` is supposed to reference, did a grep and seems to just not exist.

Reply:
Formatting `"{}", "{SYSFS_DIR}"` is intentional. This is a super lazy way to get some feedback from the compiler. The macro can make use of `static SYSFS_DIR: &str` to make a default path from it if unspecified in the attribute.

Reply:
oh ok, I was just confused because your usages didn't do that.